CHANGE: Added Windows Input tests to ensure Mouse movement works usin…

karljj1 opened the pull request:
Added a test to go with the fix in https://ono.unity3d.com/unity/unity/pull-request/102187/_/se/input/rdp-mouse-position-fix

CHANGE: Added Windows Input tests to ensure Mouse movement works using RDP.

karljj1 committed:
CHANGE: Added Windows Input tests to ensure Mouse movement works using RDP

karljj1 commented:
The tests are expected to fail until the fix lands

Rene-Damm commented on Assets/Tests/InputSystem/WindowsTests.cs:
```diff
@@ -0,0 +1,54 @@
+    [Category("WindowsInput")]
+    public IEnumerator WindowsInput_RemoteDesktopMouseMovements_AreDetected()
+    {
+        var mouse = InputSystem.GetDevice<Mouse>();
```
`Mouse.current`

karljj1 replied:
You want it moved into the IntegrationTests folder?

Rene-Damm reviewed (commented):
Since this goes end-to-end, would prefer to have this as part of `IntegrationTests`.

Also, I assume this can't land until we've landed the fix on the native side?

karljj1 committed:
Moved windows test into IntegrationTests

karljj1 commented:
Wont be able to backport this fix further than 2019.3 so these tests wont pass. Ill have a think about a solution, maybe move the test into trunk.

karljj1 committed:
Only enable the test in 2019.3 and above

The fix is not available in lower versions.

karljj1 committed:
Merge branch 'develop' into se/input/rdp-mouse-position-fix

karljj1 committed:
Updated old bintray url.

Due to error in tests
[UPM-CI-UTILS] ERROR - Usage of Bintray registry https://api.bintray.com/npm/unity/unity-npm will become obsolete by May 2020. Please, change the npm url https://api.bintray.com/npm/unity/unity-npm in /Users/builduser/build/output/Unity-Technologies/InputSystem/.yamato/upm-ci.yml for 'https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm'
